### PR TITLE
getPrivileges method using async/awaits

### PR DESCRIPTION
**Pull request by bsm5**
Tried using the async/awaits method...I think I used it properly.  Let me know what you think.

### CODE REVIEW

File: database/controllers/dbm.js [hunk 1]
```diff
@@ -206,8 +222,18 @@ module.exports = {
     //Descriptions:  This function will take a user's badge # and return
     // a JSON object with all the stations the user has been trained on and
     // can use.
-    getPrivileges(bId){
-        return true;
+    getPriv(bId){
+        return privileges.findAll({
```
Comment (danieleynis) [inline database/controllers/dbm.js]: I dont think you need two functions in this case. Pretty sure you can just do: `var results = await privileges.findAll(...)`. I dont think you need to use `async`. Then results will contain what you need and you can do the checks on that.

Comment (bsm5) [reply database/controllers/dbm.js]: Ah, okay that makes sense.  I'm taking off for the day, but can have it fixed by tomorrow morning.

Comment (danieleynis) [reply database/controllers/dbm.js]: Sounds good, thanks!

Comment (bsm5) [reply database/controllers/dbm.js]: Everything should be good now.

**Pull request merged**